src/Common/nunit/XmlHelper.cs

```diff
@@ -166,10 +165,9 @@ public static DateTime GetAttribute(this XmlNode result, string name, DateTime d
                 return defaultValue;
 
             DateTime date;
-            if (!DateTime.TryParse(dateStr, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal | DateTimeStyles.AllowWhiteSpaces, out date))
-                return defaultValue;
-
-            return date;
+            return !DateTime.TryParse(dateStr, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal | DateTimeStyles.AllowWhiteSpaces, out date) 
+                ? defaultValue 
+                : date;
```

Comment:
So change the standard

On Saturday, April 11, 2015, CharliePoole <notifications@github.com> wrote:

> In src/Common/nunit/XmlHelper.cs
> <https://github.com/nunit/nunit/pull/586#discussion_r28156662>:
>
> > @@ -166,10 +165,9 @@ public static DateTime GetAttribute(this XmlNode result, string name, DateTime d
> >                  return defaultValue;
> >
> >              DateTime date;
> > -            if (!DateTime.TryParse(dateStr, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal | DateTimeStyles.AllowWhiteSpaces, out date))
> > -                return defaultValue;
> > -
> > -            return date;
> > +            return !DateTime.TryParse(dateStr, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal | DateTimeStyles.AllowWhiteSpaces, out date)
> > +                ? defaultValue
> > +                : date;
>
> I like this better myself, but I'm pretty sure it's not in the standard.
> :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/586/files#r28156662>.
>